easybuild/easyconfigs/r/Racon/Racon-1.4.7_patch_bioparsers_zlib.patch

```diff
@@ -0,0 +1,23 @@
+diff -ru bioparser-0d66534d75b319f43ce5c59fe6548eee48f8f651.orig/CMakeLists.txt bioparser-0d66534d75b319f43ce5c59fe6548eee48f8f651/CMakeLists.txt
```

Comment:
@Darkless012 Please include a short description on top + mention author (and don't forget to update SHA256 checksum of the patch)